architecture/azure/messaging.md

```diff
@@ -0,0 +1,99 @@
+
+Azure Service Bus has two pricing tiers: [Standard and Premium](https://learn.microsoft.com/en-us/azure/service-bus-messaging/service-bus-premium-messaging). The Premium tier provides hardware-level isolation, which means each customer's workload runs on dedicated hardware. Premium tier uses _messaging units_ for scaling and pricing and guarantees predictable throughput and performance. The Standard tier offers pay-as-you-go pricing with quota-based throughput and non-guaranteed latency. Microsoft recommends running production systems on the Premium tier.
+
+:heavy_plus_sign: Pros:
+
+- Up to 100 MB message size on the Premium tier
+- Supports transactional message operations across multiple queues and topics (cross-entity transactions)
+- Scripted infrastructure deployment using the [NServiceBus ASB CLI](/transports/azure-service-bus/operational-scripting.md)
+- Supports the AMQP 1.0 protocol over TCP and WebSockets
+
+:heavy_minus_sign: Cons:
+
+- Some features are only available on the Premium tier
+- The maximum message size is 256 KB on the Standard tier
+- The maximum queue size is 80 GB on the Standard tier
+- Cross-entity transactions are limited to 100 messages
+- No cross-namespace communication
+- No emulator for local development or testing available
+
+
+[**Try the Azure Service Bus Tutorial →**](https://learn.microsoft.com/en-us/azure/service-bus-messaging/build-message-driven-apps-nservicebus?tabs=Sender)
+
+#### When to use Azure Service Bus
+
+Azure Service Bus may be considered the default messaging choice for Azure. Alternatives should be considered only when the message size limitations are not sufficient or if a more portable queueing technology is required.
+
+### Azure Queue Storage
+
+[Azure Queue Storage](https://learn.microsoft.com/en-us/azure/storage/queues/storage-queues-introduction) offer a simple, low-cost, and easy-to-use messaging service based on the Azure Storage infrastructure.
+
+:heavy_plus_sign: Pros:
+
+- Queues can contain up to 200 TB of messages
+- Cheaper than Azure Service Bus
+- Part of Azure Storage
+- Azure Storage Emulator for local development and testing
+
+:heavy_minus_sign: Cons:
+
+- The maximum message size is 64 KB
+- Higher latency due to HTTP communication
+- No cross-entity transactions
+- The maximum number of requests (e.g. message delivery) per storage account is 20,000 per second (2,000 per second for single queues)
+
+[**Try the Azure Storage Queues sample →**](/samples/azure/storage-queues/)
+
+#### When to use Azure Storage Queues
+
+Consider Azure Storage Queues when a managed message queuing technology is required for simple messaging needs or when Azure Service Bus doesn't meet critical requirements.
```

Comment:
> Consider Azure Storage Queues when a managed message queuing technology is required for simple messaging needs

That's valid for ASB too?

> or when Azure Service Bus doesn't meet critical requirements

That's…super vague.

Reply:
> That's…super vague.

intentionally, because we already state that we recommend ASB by default (in the ASB section). I guess this could be repeated.